src/RTSolution/CRTM_Active_Sensor.f90

```diff
@@ -145,9 +137,6 @@ SUBROUTINE Calculate_Cloud_Water_Density(Atm, &
    END IF
 
    dZ_m = (Atm%Height(0:n_Layers-1) - Atm%Height(1:n_Layers)) * ONE_THOUSAND
```

Comment:
are you missing the zenith angle correction in calculate cloud water density routine?

Reply:
we had a discussion about this before. The cloud water content is not scaled with respect to the zenith angle as it used dp/g to convert from mass over mass to mass over area. So no zenith angle correction is performed here because we are calculating the vertical-column integrated to density.